front_end/panels/ai_chat/ui/ConversationHistoryList.ts

```
@@ -0,0 +1,192 @@
+// Copyright 2025 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import * as ComponentHelpers from '../../../ui/components/helpers/helpers.js';
+import * as Lit from '../../../ui/lit/lit.js';
+import {createLogger} from '../core/Logger.js';
+import type {ConversationMetadata} from '../persistence/ConversationTypes.js';
+import {getConversationHistoryStyles} from './conversationHistoryStyles.js';
+
+const logger = createLogger('ConversationHistoryList');
```

ESLint

[error] 'logger' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)